docs/troubleshooting_jh.md

```diff
@@ -0,0 +1,44 @@
+# Troubleshooting JupyterHub for Data Scientists
```

Comment:
Overall Comment: It might be good idea to split this doc into a couple of sections for example: 1. a list of specific known issues and solutions for how to fix them, like "kernel is crashing, must restart pod with more resources". 2. A more general section on the available monitoring techniques for RAM, PVC, CPU, and Logs that are available letting folks know where they can go to diagnose different issues. WYDT?

Reply:
That sounds good, although I intended the doc to only talk about the second section since @hemajv mentioned that there are [runbooks](https://github.com/operate-first/blueprint/tree/main/docs/userstories/jupyterhub) for known issues. I still think there's value in adding known issues with screenshots to the user document. If that's what is reasonable, we should update the original [google doc](https://docs.google.com/document/d/1Y83HEUQbwilc1K4gpVkt79VXH3T02TSzTFhPwvMVXSY/edit)  and then with all the additional suggestions we can change this PR. I'll take most suggestions from here and work on it but feel free to add more known issues, and content.